src/ManageDatesView.elm

```diff
@@ -33,44 +33,54 @@ view model =
-                            [ input
-                                [ class "form-control pikaday-input"
-                                , id ("pikaday-instance-" ++ (toString i))
-                                , value (displayDateWithoutTime d)
-                                , disableInput d
+                        div [ class "form__question-section form__question-section--table" ]
+                            [ div [ class "form__question-sub-section form__question-sub-section--table" ]
+                                [ label [ class "form__label" ]
+                                    [ text "Date" ]
+                                , input
+                                    [ class "form__input  pikaday-input"
+                                    , id ("pikaday-instance-" ++ (toString i))
+                                    , value (displayDateWithoutTime d)
+                                    , disableInput d
+                                    ]
+                                    []
                                 ]
-                                []
-                            , button
-                                [ onClick (DeleteDate d)
-                                , style [ ( "margin-left", "0.2rem" ) ]
-                                , disableInput d
+                            , div [ class "form__question-sub-section form__question-sub-section--table form__question-sub-section__button" ]
+                                [ button
+                                    [ onClick (DeleteDate d)
+                                    , disableInput d
+                                    , class "button button--secondary icon icon--bin"
+                                    ]
+                                    []
                                 ]
-                                [ text "Delete" ]
                             ]
                     )
 
         column1 =
-            div [ class "form-group" ]
-                [ div [ class "input-group" ]
+            div []
+                [ span [ class "form__hint" ]
+                    [ text "You will be unable to change any dates that have sessions" ]
+                , div []
                     datesInputs
-                , div [ style [ ( "margin-top", "1rem" ) ] ]
+                , div []
                     [ button
-                        [ class "btn btn-default"
+                        [ class "button button--tertiary"
                         , id "add-new-date-btn"
                         , type_ "button"
                         , onClick (GetDateAndThenAddDate <| toString <| List.length model.pickedDates)
                         ]
                         [ text "Add New Date" ]
                     ]
-                , div [ style [ ( "margin-top", "1rem" ) ] ]
-                    [ button [ class "btn btn-default", id "save-dates-btn", type_ "button" ]
-                        [ text "Save Dates" ]
+                , div []
```

Comment:
please can this div have the classes `.bar` and `.bar--button`

![screen shot 2017-03-13 at 12 48 06](https://cloud.githubusercontent.com/assets/7805691/23854893/699e4fb8-07eb-11e7-9368-61cdf84b26e4.png)